arduino/smartcar/smartcar.ino

```diff
@@ -41,15 +40,7 @@ void setup() {
     mqtt.subscribe("/smartcar/control/#", 1);
     mqtt.onMessage([](String topic, String message) {
       if (topic == "/smartcar/control/throttle") {
-        if (canDriveForward && message.toInt() >= 0){
```

Comment:
I believe this is an old version of our arduino sketch, but canDriveForward and canDriveBackwards needs to be in the sketch for the obstacle avoidance